functions/src/api/v1/reminders/editReminder.ts

```diff
@@ -99,9 +95,9 @@ export const editReminder = wrap(async (req: Request, res: Response) => {
         dataToEdit.hostName = hostName
     }
 
-    await updateReminderDb(dataToEdit)
+    await ReminderDao.update(dataToEdit)
 
-    const reminder = await getReminderDb(reminderId)
+    const reminder = await ReminderDao.get(reminderId)
 
     res.send({
```

Comment:
it's not consistent with swagger status code definition. https://github.com/Instant-Visio/InstantVisio-WebApp/blob/1a7d974bab4ef0366cd884265b29fadfc5ad7b17/functions/src/api/v1/reminders/editReminder.ts#L44 You should change swagger definition to `200`